### ISSUE TITLE
BLEA 2.0.0 does not be supported organization trails in Control Tower landing zone version 3.0

### ISSUE THREAD

**Issue by toshiyuki-tanaka-673** [aws-samples/baseline-environment-on-aws#61]
# Describe issue

BLEA 2.0.0 does not be supported organization trails in Control Tower landing zone version 3.0

### AWS Control Tower landing zone version 3.0 includes the following updates:

- The option to choose organization-level AWS CloudTrail trails
- We no longer create the CloudWatch log group aws-controltower/CloudTrailLogs in each enrolled account. Previously, we created these in each account for its account trail. With organization trails, we only create one in the management account.
https://docs.aws.amazon.com/controltower/latest/userguide/2022-all.html#version-3.0

### BLEA 2.0.0 (latest) assumes the previous ControlTower version specification and is written as follows.

- CloudWatch LogGroup Name for CloudTrail - Created by ControlTower for each account
https://github.com/aws-samples/baseline-environment-on-aws/blob/9bb15dfd4ed282cafd9b6e1fda9fd096b1bcaded/usecases/base-ct-guest/bin/blea-base-ct-guest.ts
```ts
// Security Alarms
// !!! Need to setup SecurityHub, GuardDuty manually on Organizations Management account
// AWS Config and CloudTrail are set up by ControlTower

// CloudWatch LogGroup Name for CloudTrail - Created by ControlTower for each account
const cloudTrailLogGroupName = 'aws-controltower/CloudTrailLogs';

const secAlarm = new BLEASecurityAlarmStack(app, `${pjPrefix}-SecurityAlarm`, {
  notifyEmail: envVals['securityNotifyEmail'],
  cloudTrailLogGroupName: cloudTrailLogGroupName,
  env: getProcEnv(),
});
```

## A reproducible test case or series of steps
Control Tower landing zone version 3.0
Enable the option to choose organization-level AWS CloudTrail trails.
[Deploy to Multiaccount environment](https://github.com/aws-samples/baseline-environment-on-aws/blob/main/doc/DeployToControlTower.md)

##  The version of our code being used
BLEA v.2.0.0
Control Tower landing zone version 3.0

## Any modifications you've made relevant to the bug
none

## Anything unusual about your environment or deployment
none

**Comment by konokenj** [comment]
Thank you for your reporting. We're planning to fix this.

**Comment by ohmurayu** [comment]
Thank you for feedback. We has resolved this with #70 .
We changed default environment to LZ3.0 with CloudTrail support, and add document so you can handle LZ 2.9 and before.
